lib/features/thread/presentation/thread_controller.dart

```diff
@@ -140,6 +158,12 @@ class ThreadController extends BaseController {
 
   @override
   void onClose() {
+    _searchDebounce?.cancel();
+    mailboxDashBoardController.selectedMailbox.close();
```

Comment:
We already use clear `Worker` so no need to close `RxObject`

Reply:
why you resolve that comment without any fix or talk back?

Reply:
@hoangdat  Shall I use worker or keep using rxobject?

Reply:
@johnnguyenit39 You are getting it wrong. I mean when you want to listen to `RxObject` object you don't need to use `RxObect.listen()` method instead you use `ever` it returns a `worker`. Ex: `ever(rxobject, (event) => {})`

Reply:
In your current code, there are still lines of `close listen` code of `RxObject` that you should delete. Because I replaced it with `clear worker`. You should rebase the code on the master branch to see that.